tests/cpu/transforms/LookTransform_tests.cpp

```diff
@@ -559,3 +562,191 @@ environment: { FILE1: cdl_test1.cc, FILE2: cdl_test1.cc }
+    OCIO_CHECK_ASSERT(lg);
+    OCIO_CHECK_EQUAL(lg->getDirection(), OCIO::TRANSFORM_DIR_INVERSE);
+    op = ops[8]; // raw to vd
+    OCIO_REQUIRE_EQUAL(op->data()->getType(), OCIO::OpData::GammaType);
+    auto gm = OCIO_DYNAMIC_POINTER_CAST<const OCIO::GammaOpData>(op->data());
+    OCIO_CHECK_ASSERT(gm);
+    OCIO_CHECK_EQUAL(gm->getDirection(), OCIO::TRANSFORM_DIR_INVERSE);
+    op = ops[9];
+    OCIO_CHECK_ASSERT(op->isNoOpType());
+    op = ops[10];
+    OCIO_CHECK_ASSERT(op->isNoOpType());
+
+    // Test in inverse direction.
+    ops.clear();
+    OCIO_CHECK_NO_THROW(BuildColorSpaceOps(ops, *config, config->getCurrentContext(),
+                                           dstColorSpace, srcColorSpace, false));
+    OCIO_REQUIRE_EQUAL(ops.size(), 11);
+    OCIO_CHECK_NO_THROW(ops.validate());
+    op = ops[0];
+    OCIO_CHECK_ASSERT(op->isNoOpType());
+    op = ops[1];
+    OCIO_CHECK_ASSERT(op->isNoOpType());
+    op = ops[2]; // vd to raw
+    OCIO_REQUIRE_EQUAL(op->data()->getType(), OCIO::OpData::GammaType);
+    gm = OCIO_DYNAMIC_POINTER_CAST<const OCIO::GammaOpData>(op->data());
+    OCIO_CHECK_ASSERT(gm);
+    OCIO_CHECK_EQUAL(gm->getDirection(), OCIO::TRANSFORM_DIR_FORWARD);
+    op = ops[3]; // raw to log
+    OCIO_REQUIRE_EQUAL(op->data()->getType(), OCIO::OpData::LogType);
+    lg = OCIO_DYNAMIC_POINTER_CAST<const OCIO::LogOpData>(op->data());
+    OCIO_CHECK_ASSERT(lg);
+    OCIO_CHECK_EQUAL(lg->getDirection(), OCIO::TRANSFORM_DIR_FORWARD);
+    op = ops[4];
+    OCIO_CHECK_ASSERT(op->isNoOpType());
+    op = ops[5];
+    OCIO_CHECK_ASSERT(op->isNoOpType());
+    op = ops[6]; // look
+    OCIO_REQUIRE_EQUAL(op->data()->getType(), OCIO::OpData::CDLType);
+    cdl = OCIO_DYNAMIC_POINTER_CAST<const OCIO::CDLOpData>(op->data());
+    OCIO_CHECK_ASSERT(cdl);
+    OCIO_CHECK_EQUAL(cdl->getDirection(), OCIO::TRANSFORM_DIR_INVERSE);
+    op = ops[7];
+    OCIO_CHECK_ASSERT(op->isNoOpType());
+    op = ops[8]; // log to raw
+    OCIO_REQUIRE_EQUAL(op->data()->getType(), OCIO::OpData::LogType);
+    lg = OCIO_DYNAMIC_POINTER_CAST<const OCIO::LogOpData>(op->data());
+    OCIO_CHECK_ASSERT(lg);
+    OCIO_CHECK_EQUAL(lg->getDirection(), OCIO::TRANSFORM_DIR_INVERSE);
+    op = ops[9];
+    OCIO_CHECK_ASSERT(op->isNoOpType());
+    op = ops[10];
+    OCIO_CHECK_ASSERT(op->isNoOpType());
+
+    // Compare to vd_graded_2 that should be identical (only diference is that
+    // it's defined using to_reference instead of from_reference)
+    OCIO::ConstColorSpaceRcPtr dstColorSpace2;
+    OCIO_CHECK_NO_THROW(dstColorSpace2 = config->getColorSpace("vd_graded_2"));
+
+    OCIO::OpRcPtrVec ops2;
+    OCIO_CHECK_NO_THROW(BuildColorSpaceOps(ops2, *config, config->getCurrentContext(),
+                                           dstColorSpace2, srcColorSpace, false));
+    OCIO_REQUIRE_EQUAL(ops2.size(), 11);
+    OCIO_CHECK_NO_THROW(ops2.validate());
+    op = ops2[0];
+    OCIO_CHECK_ASSERT(op->isNoOpType());
```

Comment:
Looks like OpRcPtrVec does not have an op==, which could shorten the test and make it easier to read.  Maybe Patrick will have a suggestion for how to write it more compactly.  But what you have here looks correct, so no objection to leaving it as is.

Reply:
Waiting for Patrick feedback, here is a shorter version I came up with just in case:
```
OCIO_REQUIRE_EQUAL(ops2.size(), ops.size());
OCIO_CHECK_NO_THROW(ops2.validate());
for (int i = 0; i < ops2.size(); ++i)
{
   OCIO::ConstOpRcPtr op = ops[i];
   OCIO::ConstOpRcPtr op2 = ops2[i];
   OCIO_REQUIRE_ASSERT(*op->data() == *op2->data());
}
```

Reply:
Updated the test